android/titanium/build.gradle

```diff
@@ -279,6 +279,15 @@ dependencies {
 
 	// Reference all local JAR file dependencies.
 	implementation fileTree(dir: 'lib', include: ['*.jar'])
+
+	// CameraX
+  def camerax_version = '1.1.0'
+  implementation "androidx.camera:camera-core:$camerax_version"
+  implementation "androidx.camera:camera-camera2:$camerax_version"
+  implementation "androidx.camera:camera-video:$camerax_version"
+  implementation "androidx.camera:camera-lifecycle:$camerax_version"
+  implementation "androidx.camera:camera-view:$camerax_version"
+  implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
```

Comment:
2.1.4 is available: https://developer.android.com/jetpack/androidx/releases/constraintlayout